errata_tool/erratum.py

```diff
@@ -232,8 +232,10 @@ def _fetch(self, errata_id):
             # Actual ship date (if in SHIPPED_LIVE)
             if self.errata_state in ('SHIPPED_LIVE'):
                 d = erratum['actual_ship_date']
-                d = time.strptime(str(d), '%Y-%m-%dT%H:%M:%SZ')
-                self.ship_date = time.strftime('%Y-%b-%d', d)
+                # Could be None. e.g. advisory 43686
+                if d:
+                    d = time.strptime(str(d), '%Y-%m-%dT%H:%M:%SZ')
+                    self.ship_date = time.strftime('%Y-%b-%d', d)
```

Comment:
Is there a logical `else` condition for setting `ship_date` if no date is found? I imagine other tooling would expect `self.ship_date` to exist in some manner....

Reply:
I didn't check that. This at least can prevent elliott from crashing...

Reply:
ship_date is already set to None, so this should be fine, would like to see a unittest exercising this failure mode but that can be a followup patch.